docs/ref/forms/models.txt

```diff
@@ -81,3 +81,40 @@ Model Form API reference. For introductory material about model forms, see the
     the ``parent_model``, you must specify a ``fk_name``.
 
     See :ref:`inline-formsets` for example usage.
+
+``ModelForm``
```

Comment:
Unrelated documentation to the fix should go in a separate commit.

Reply:
Is it unrelated? I was asked to add documentation for this feature, and since there was no API documentation for ModelForm yet, I included it.